src/App.js

```diff
@@ -10,10 +12,35 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+export class App extends React.PureComponent {
+  state = {
+    selectedTabId: 'tab-2',
```

Comment:
```suggestion
    selectedTabId: 'tab-1',
```

За замовчуванням має стояти перша таба